python/semantic_kernel/connectors/memory/azure_cosmosdb_no_sql/utils.py

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from collections.abc import Sequence
+from typing import Any
+
+from semantic_kernel.connectors.memory.azure_cosmosdb_no_sql.azure_cosmos_db_no_sql_composite_key import (
+    AzureCosmosDBNoSQLCompositeKey,
+)
+from semantic_kernel.data.const import DistanceFunction, IndexKind
+from semantic_kernel.data.vector_store_model_definition import VectorStoreRecordDefinition
+from semantic_kernel.data.vector_store_record_fields import VectorStoreRecordDataField, VectorStoreRecordVectorField
+
+
+def to_vector_index_policy_type(index_kind: IndexKind) -> str:
+    """Converts the index kind to the vector index policy type for Azure Cosmos DB NoSQL container.
+
+    Depending on the index kind, the maximum number of dimensions may be limited:
+    https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/vector-search#vector-indexing-policies
+
+    Args:
+        index_kind (IndexKind): The index kind.
+
+    Returns:
+        str: The vector index policy type.
+    """
+    match index_kind:
```

Comment:
I always prefer dicts for these kinds of mappings, not sure if there are mayor improvements with either approach, just saying.